modules/migration-rsync-manually-overriding.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: PROCEDURE
+[id="migration-rsync-manually-overriding_{context}"]
+== Manually overriding default non-root operation for data transfer
+
+Although running Rsync pods as non-root user works in most cases, data transfer might fail when when you run workloads as root user on the source side. {mtc-short} provides two ways to manually override default non-root operation for data transfer: configuring all migrations to run an Rsync pod as root on the destination cluster for all migrations or to run an Rsync pod as root on the destination cluster per migration.
```

Comment:
Nit: Use a bulleted list for clarity

```suggestion
Although running Rsync pods as non-root user works in most cases, data transfer might fail when when you run workloads as root user on the source side. {mtc-short} provides two ways to manually override default non-root operation for data transfer: 
* Configure all migrations to run an Rsync pod as root on the destination cluster for all migrations
* Run an Rsync pod as root on the destination cluster per migration
```

Reply:
@aireilly I see what happened here -- I copy-pasted the upstream destription into one file, which I at frst thought would be a single module, then I edited the text. At a certain point it became obvious that this this file should be broken into modules, which I did, working from my large moule. I should've simply gotten rid of teh latge module after (note it's not called by any assembly). I've deleted it, but made the changes you suggest in the componenet modules that I created for thsi PR.

Reply:
Bulleted list -- great idea. Done.